bin/stringsCompletenessTest.py

```diff
@@ -0,0 +1,140 @@
+                    # print(property_key)
+                    l10n_properties[property_key] = 0
+                    # print(l10n_properties[property_key])
+
+    # directories to look in
+    def_code_dirs = ["add-on/chrome", "add-on/panels/js", "add-on/panels/vendor", "shared/js", "standalone/content/js"]
+    # regex string getter methods to look for
+    pattern = re.compile('_getString\(\"(.*)\"\)|mozL10n\.get\(\"(.*)\"\)')
+
+    # iterate search through files in code directories
+    props_found = 0
+    for code_dir in def_code_dirs:
+        os.chdir(loop_root_dir)
+        print("Current directory: ", os.getcwd())
+        print("code_dir", code_dir)
+        os.chdir(code_dir)
+        print("Current directory: ", os.getcwd())
+        for fname in multiple_file_types("*.js", "*.jsx", "*.jsm"):
+            file_path = os.path.join(os.getcwd(), fname)
+            print(fname)
+            print("fname", fname)
+            print("file_path", file_path)
+            with io.open(file_path, "r", encoding="UTF-8") as code_file:
+                for line in code_file:  # process line by line
+                    # print(line)
+                    for match in re.finditer(pattern, line):
+                        print(pattern)
+                        print(line,)
+                        print('Group 1: %s' % (match.group(1)))
+                        print('Group 2: %s' % (match.group(2)))
+                        if match.group(1):
+                            code_key = match.group(1).encode("utf-8")
+                            print("code_key", code_key)
+                            if code_key in l10n_properties:
+                                print("FOUND ", code_key)
+                                l10n_properties[code_key] = 1
+                                props_found += 1
+                                # print(l10n_properties[code_key])
+                        if match.group(2):
+                            code_key = match.group(2).encode("utf-8")
+                            print("code_key", code_key)
+                            if code_key in l10n_properties:
+                                print("FOUND ", code_key)
+                                l10n_properties[code_key] = 1
+                                props_found += 1
+                                # print(l10n_properties[code_key])
+                # en_us_locale_files = os.listdir(l10n_code_src_dir)
+
+    # TODO - find all missing strings task and print out line
+    # further searches and tests for larger insight into string use cases and actual unused strings
+    # gather other useful information
+    # get list of = "literal strings in the code";
+    # Iterate back over code with patterns from not found
+    # or unused strings to find literal strings used and where (line).
+
+    # print report
+    # print (l10n_properties.keys())
+
+    for prop in l10n_properties:
+        print(prop,)
+        print(l10n_properties.get(prop))
```

Comment:
I think the initial report should just print out two things: 1) unused strings, 2) strings not in the locale files.